.env.sample

```diff
@@ -0,0 +1,8 @@
+SECRET_KEY==xti712i87pm=a_wyq$chj8v)zqzzp))q1*5!padcx1yhx=%^)
```

Comment:
Secret key shouldn't be visible, right?

Reply:
sample mein rakha hai so that people can copy on their local repo.